lib/parser/concurrentStatementParser.ts

```diff
@@ -201,9 +202,8 @@ export class ConcurrentStatementParser extends ParserBase {
       const read = new ORead(assignment, readToken);
       assignment.labelLinks.push(read);
       this.parent.statements.push(assignment);
-    } else if (nextToken.getLText() === 'assert' && allowedStatements.includes(ConcurrentStatementTypes.Assert)) {
-      this.consumeToken();
-      this.advanceSemicolon();
+    } else if ((nextToken.getLText() === 'assert' || nextToken.getLText() === 'postponed') && allowedStatements.includes(ConcurrentStatementTypes.Assert)) {
```

Comment:
processes, concurrent procedure calls and concurrent signal assignments can also be postponed (refer 11.3, 11.4 and 11.6)